cockatrice/src/dlg_settings.cpp

```diff
@@ -648,6 +659,8 @@ void MessagesSettingsPage::retranslateUi()
     chatMentionCheckBox.setText(tr("Enable chat mentions ('@yourusername' in chat log will be highlighted)"));
     messageShortcuts->setTitle(tr("In-game message macros"));
     ignoreUnregUsersMainChat.setText(tr("Ignore unregistered users in main chat"));
+    mentionColorLabel.setText(tr("Username/Mention color:"));
+    invertMentionForeground.setText(tr("Invert mention text color (white)"));
```

Comment:
I'd like to group this option under the mention enabling checkbox rather than under it's own heading. And the unreg ignore option looks like it's under the chat mention heading